godockerize.go

```diff
@@ -149,10 +148,17 @@ func doBuild(c *cli.Context) error {
 
 	for _, pkg := range install {
 		if strings.HasSuffix(pkg, "@edge") {
-			fmt.Fprintf(&dockerfile, "  RUN echo -e \"@edge http://dl-cdn.alpinelinux.org/alpine/edge/main\\n@edge http://dl-cdn.alpinelinux.org/alpine/edge/community\" >> /etc/apk/repositories\n")
+			fmt.Fprintf(&dockerfile, `  RUN echo -e "http://dl-cdn.alpinelinux.org/alpine/edge/main\n" >> /etc/apk/repositories
+  RUN echo -e "http://dl-cdn.alpinelinux.org/alpine/edge/community\n" >> /etc/apk/repositories
```

Comment:
We should probably keep this as a single `RUN` command in the dockerfile. The reason to do it that way is because each RUN command introduces a new layer:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
Oh, right, ugh.

Reply:
Done